src/Components/Publishing/Sections/ImageCollection.tsx

```diff
@@ -20,83 +20,82 @@ interface ImageCollectionProps {
+  renderImages(dimensions) {
+    const {
+      gutter,
+      images,
+      sectionLayout,
+      size: {
+        width
+      }
+    } = this.props
 
-  return (
-    <ImageCollectionContainer>
-      {renderedImages}
-    </ImageCollectionContainer>
-  )
-}
+    const renderedImages = images.map((image, i) => {
+      const url = image.url || image.image
 
-ImageCollectionComponent.defaultProps = {
-  size: {
-    width: 680,
-  },
-}
+      let imageSize
+      if (width <= 600 || dimensions.length === 1) {
+        imageSize = {}
+      } else {
+        imageSize = find(dimensions, ["__id", url])
+      }
 
-function renderImages(images, dimensions, gutter, sectionLayout, width) {
-  const renderedImages = images.map((image, i) => {
-    const url = image.url || image.image
+      let renderedImage
+      if (image.type === "image") {
+        renderedImage = (
+          <Image
+            image={image}
+            sectionLayout={sectionLayout}
+            width={imageSize.width}
+            height={imageSize.height}
+          />
+        )
+      } else if (image.type === "artwork") {
+        renderedImage = (
+          <Artwork
+            artwork={image}
+            sectionLayout={sectionLayout}
+            width={imageSize.width}
+            height={imageSize.height}
+          />
+        )
+      } else {
+        return false
+      }
 
-    let imageSize
-    if (width <= 600 || dimensions.length === 1) {
-      imageSize = {}
-    } else {
-      imageSize = find(dimensions, ["__id", url])
-    }
+      const margin = i === dimensions.length - 1 ? 0 : gutter
 
-    let renderedImage
-    if (image.type === "image") {
-      renderedImage = (
-        <Image
-          image={image}
-          sectionLayout={sectionLayout}
+      return (
+        <ImageCollectionItem
+          key={i}
+          margin={margin}
           width={imageSize.width}
-          height={imageSize.height}
-        />
+        >
+          {renderedImage}
+        </ImageCollectionItem>
       )
-    } else if (image.type === "artwork") {
-      renderedImage = (
-        <Artwork
-          artwork={image}
-          sectionLayout={sectionLayout}
-          width={imageSize.width}
-          height={imageSize.height}
-        />
-      )
-    } else {
-      return false
-    }
+    })
+    return renderedImages
+  }
 
-    const margin = i === dimensions.length - 1 ? 0 : gutter
+  render() {
+    const { gutter, images, size, targetHeight } = this.props
+    const dimensions = fillwidthDimensions(images, size.width, gutter, targetHeight)
+    const renderedImages = this.renderImages(dimensions)
 
     return (
-      <ImageCollectionItem
-        key={i}
-        margin={margin}
-        width={imageSize.width}
-      >
-        {renderedImage}
-      </ImageCollectionItem>
+      <ImageCollectionContainer>
+        {renderedImages}
+      </ImageCollectionContainer>
     )
-  })
-  return renderedImages
+  }
```

Comment:
DRYed this and the below up a bit so we don't have to pass stuff from function to function to function. It's less error prone and maintainable to just pluck from props.